hazelcast/src/main/java/com/hazelcast/client/impl/protocol/util/ClientProtocolOutput.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2014 Real Logic Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.client.impl.protocol.util;
+
+/**
+ * Abstraction over a range of buffer types that allows fields to be written in client protocol.
+ * All methods use LITTLE_ENDIAN as byte order.
```

Comment:
`All methods use LITTLE_ENDIAN as byte order` seems a javaDoc for an implementation.